content/sensu-core/1.0/guides/scaling-overview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Scaling Sensu - Overview"
+linkTitle: "Scaling Overview"
+product: "Sensu Core"
+description: "Overview of strategies and best practices for scaling Sensu"
+version: "1.0"
+weight: 11
+next: ../scaling-sensu-components

```

Comment:
I think this is in the 1.4 one as well, but do we want these next pages as they are 404's until this content is ready? I've seen it used elsewhere, good reminder/ensure it works when that page is there, but might be a bad user experience.